A.Anderson W.I.P.

AAnderson21 opened the pull request.

AndjDavis commented:
Looks great so far, everything is settling into place.  Submit another PR when you're done so I can see your finished work.   As a side note, I would suggest to commit a lot more and with a more detailed commit message.  Something along the lines of 'finished the Header, the about, and the features sections' gives you a lot better idea of what you worked on for that commit.  good job!

AAnderson21 commented:
Updated Version of Project 1.
Cleaned up page.
Inserted flexbox for "box" properties
Still a work in progress.
Need walk through on missing pieces tomorrow if possible.

*Also when you said "commit a lot more". What were you referring too?
